src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,44 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+
+interface TodoModalProps {
+  handleCloseModal: () => void;
+  selectedUserId: number;
+  selectedTodo: Todo | null;
+}
+export const TodoModal: React.FC<TodoModalProps> = ({
+  handleCloseModal,
+  selectedUserId,
+  selectedTodo,
+}) => {
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Consider setting `setIsLoading(true)` at the beginning of the `handleLoadUserInfo` function to ensure the loading state is correctly managed when fetching user data. This will help in scenarios where the modal is reopened and data needs to be fetched again.